README.md

```diff
@@ -13,34 +13,34 @@ This project contains the abstract layer for APIMatic's core library and Apimati
-| [`DynamicType`](./src/main/java/io/apimatic/coreinterfaces/http/response/DynamicType.java)          | To get the properties from the instance of Dynamic Response Type                |
-| [`ApiResponseType`](./src/main/java/io/apimatic/coreinterfaces/http/response/ApiResponseType.java)          | To get the properties from the instance of ApiResponseType                |
-| [`CompatibilityFactory`](./src/main/java/io/apimatic/coreinterfaces/compatibility\CompatibilityFactory.java)                  | Holds required converter methods to create SDK classes             |
-| [`CoreFileWrapper`](./src/main/java/io/apimatic/coreinterfaces/type/CoreFileWrapper.java)                  | To wrap file and contentType to be sent as part of a HTTP request.            |
-| [`HttpApiException`](./src/main/java/io/apimatic/coreinterfaces/type/HttpApiException.java)                  | Default interface for all the exception models in SDK              |
-| [`ApiLogger`](./src/main/java/io/apimatic/coreinterfaces/logger/ApiLogger.java)                  | Interface for Loggin Api Calls.              |
-| [`Logger`](./src/main/java/io/apimatic/coreinterfaces/logger/Logger.java)                  | Interface for logging at different levels.              |
-| [`LoggingConfiguration`](./src/main/java/io/apimatic/coreinterfaces/logger/configuration/LoggingConfiguration.java)                  | To hold logging configuration              |
-| [`LoggingOptions`](./src/main/java/io/apimatic/coreinterfaces/logger/configuration/LoggingOptions.java)                  | To hold base options for logging configuration              |
-| [`RequestLoggingOptions`](./src/main/java/io/apimatic/coreinterfaces/logger/configuration/RequestLoggingOptions.java)                  | To hold Request options for logging configuration              |
-| [`ResponseLoggingOptions`](./src/main/java/io/apimatic/coreinterfaces/logger/configuration/ResponseLoggingOptions.java)                  | To hold Response options for logging configuration              |
-| [`Deserializer`](./src/main/java/io/apimatic/coreinterfaces/type/functional/Deserializer.java)                  | Functional interface to  apply the deserialization function           |
-| [`ExceptionCreator`](./src/main/java/io/apimatic/coreinterfaces/type/functional/ExceptionCreator.java)                  | Functional interface to  create the SDK exception             |
-| [`Serializer`](./src/main/java/io/apimatic/coreinterfaces/type/functional/Serializer.java)                  | Functional interface to  apply the serialization function              |
-| [`ContextInitializer`](./src/main/java/io/apimatic/coreinterfaces/type/functional/ContextInitializer.java)                  | Functional Interface to apply the context initialization function for the response models             |
+| Name                                                                    | Description                                                                            |
```

Comment:
The removed classes/enums should be removed from the ReadMe as well. If you see the `LoggingLevel` and `LoggingLevelType` at the bottom of the ReadMe file.

Reply:
Enums removed from the readme file.